ruma-federation-api/src/keys/claim_keys/v1.rs

```diff
@@ -0,0 +1,56 @@
+//! [POST
+//! /_matrix/federation/v1/user/keys/claim](https://matrix.org/docs/spec/server_server/r0.1.4#post-matrix-federation-v1-user-keys-claim)
+
+use std::collections::BTreeMap;
+
+use ruma_api::ruma_api;
+use ruma_identifiers::{DeviceId, DeviceKeyAlgorithm, DeviceKeyId, UserId};
+use serde::{Deserialize, Serialize};
+
+type KeyClaim = BTreeMap<UserId, BTreeMap<Box<DeviceId>, DeviceKeyAlgorithm>>;
+type KeyResponse = BTreeMap<UserId, BTreeMap<Box<DeviceId>, BTreeMap<DeviceKeyId, KeyObject>>>;
```

Comment:
These need to be public so users can see their definition in the docs.

Reply:
Also, I think these should be named the same way you described them in the constructor docs – `OneTimeKeyClaims` & `OneTimeKeys`.